go/login/server/keymanager_test.go

```diff
@@ -0,0 +1,225 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package server
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/google/glome/go/glome"
+)
+
+func tests() map[string]*KeyManager {
```

Comment:
I don't understand why we should be using these 3 with each test. Consider using an empty KeyManager for basic tests and a prefilled one where you want to test that e.g. unrelated existing keys are not dropped on `Add`, but are on `DropAllReplace`.

Reply:
SGTM!